.github/actions/run_local_pytest/action.yml

```diff
@@ -0,0 +1,35 @@
+name: 'Run Local Pytest'
+description: 'Installs Python dependencies and runs Pytest against locally built ArcticDB binaries'
+inputs:
+  job_type:          {default: '', type: string, description: Specifies which steps should be executed}
+  build_type:        {default: 'release', type: string, description: Specifies whether the build that we are testing is release or debug}
+  threads:           {default: '1', type: string, description: Specifies the number of threads that will be used for tests }
+  fast_tests_only:   {default: '0', type: string, description: Specifies whether we want to run only the fast tests or not }
+  other_params:      {default: '', type: string, description: Other params that should be passed to the python command }
+runs:
+  using: "composite"
+  steps:
+    - name: Install local dependencies with pip
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip
+        ARCTIC_CMAKE_PRESET=skip pip install -ve .[Testing]
+
+    - name: Run local pytests
+      if: inputs.job_type == 'run'
+      shell: bash
+      run: |
+        yum install nodejs npm -y
+        npm config set strict-ssl false
+        npm install -g azurite
+        
+        cd python
+        ln -s ../cpp/out/linux-${{ inputs.build_type }}-build/arcticdb/arcticdb_ext.cpython-36m-x86_64-linux-gnu.so
+        export ARCTICDB_RAND_SEED=$RANDOM
+        python ${{inputs.other_params}} -m pytest -n ${{ inputs.threads }} tests
```

Comment:
Why not use the built-in `-n auto`?

Reply:
On linux, I want to pass -n 0, which disables xdist entirely because of the mongo tests.

Reply:
That's fine. I'll address this once my new Mongo fixture is merged.